catalog/confluence/catalog-confluence-source/src/test/java/org/codice/ddf/confluence/source/ConfluenceInputTransformerTest.java

```diff
@@ -190,8 +192,10 @@ private void validatePageMetacard(
     assertThat(
         mcard.getAttribute(Associations.EXTERNAL).getValues(),
         contains("https://codice.atlassian.net/wiki/display/DDF/Formatting+Source+Code"));
-    assertThat(mcard.getCreatedDate(), is(getDate("2013-09-18T14:50:42.616-07:00")));
-    assertThat(mcard.getModifiedDate(), is(getDate("2015-06-16T19:21:39.141-07:00")));
+    assertThat(
+        mcard.getAttribute(Core.CREATED).getValue(), is(getDate("2013-09-18T14:50:42.616-07:00")));
+    assertThat(
+        mcard.getAttribute(Core.MODIFIED).getValue(), is(getDate("2015-06-16T19:21:39.141-07:00")));
```

Comment:
❓@ethantmanns I'm a tad confused why the above methods are deprecated if, according to these test assertions, they are equivalent to directly going through the taxonomy.

Reply:
❓Also I'm a tad confused why no additional code needs to be introduced if code depends on the deprecated semantics of the metacard date methods.

Reply:
The API documentation is wrong because those taxonomy terms no longer represent what the method claims to return

Reply:
It claims to return the metacard date, but it returns the resource date. This is because that same attribute name previously represented the metacard date, but now represents the resource date.